cfme/services/catalogs/catalog.py

```diff
@@ -98,6 +99,7 @@ def update(self, updates):
         else:
             view.cancel_button.click()
         view = self.create_view(DetailsCatalogView, override=updates)
+        wait_for(lambda: view.is_displayed, delay=0.5, num_sec=2)
```

Comment:
If the 0.2s default delay in wt.core.View.wait_displayed will work here, please use `view.wait_displayed` instead of wrapping it. Default timeout is 10s for wait_displayed.

Reply:
I've opened a wt.core enhancement to allow you to pass that delay to `wait_displayed`, and to assert against its return (so you can remove L101/103 below)

Reply:
OK will do mshriver